.eslintrc.cjs

```diff
@@ -0,0 +1,194 @@
+module.exports = {
+	"root": true,
+	"parser": "@typescript-eslint/parser",
+	"parserOptions": {
+		"sourceType": "module",
+		"ecmaFeatures": {
+			"impliedStrict": true,
+		},
+	},
+	"env": {
+		"browser": true,
+		"es2020": true,
+	},
+	"plugins": ["@typescript-eslint", "@stylistic"],
+	"rules": {
+		// ESLint rules: https://eslint.org/docs/latest/rules/
+		// Based off of: https://github.com/eslint/eslint/blob/v8.54.0/packages/js/src/configs/eslint-recommended.js
+		"constructor-super": 1, // Require `super()` calls in constructors
+		"curly": 1, // Enforce consistent brace style for all control statements
+		"for-direction": 1, // Enforce “for” loop update clause moving the counter in the right direction
+		"getter-return": 1, // Enforce `return` statements in getters
+		"no-async-promise-executor": 1, // Disallow using an async function as a Promise executor
+		"no-case-declarations": 1, // Disallow lexical declarations in case clauses
+		"no-class-assign": 1, // Disallow reassigning class members
+		"no-compare-neg-zero": 1, // Disallow comparing against -0
+		"no-const-assign": 1, // Disallow reassigning `const` variables
+		"no-constant-condition": 1, // Disallow constant expressions in conditions
+		"no-control-regex": 1, // Disallow control characters in regular expressions
+		"no-debugger": 1, // Disallow the use of `debugger`
+		"no-delete-var": 1, // Disallow deleting variables
+		"no-dupe-args": 1, // Disallow duplicate arguments in `function` definitions
+		"no-dupe-class-members": 1, // Disallow duplicate class members
+		"no-dupe-else-if": 1, // Disallow duplicate conditions in if-else-if chains
+		"no-dupe-keys": 1, // Disallow duplicate keys in object literals
+		"no-duplicate-case": 1, // Disallow duplicate case labels
+		"no-empty-character-class": 1, // Disallow empty character classes in regular expressions
+		"no-empty-pattern": 1, // Disallow empty destructuring patterns
+		"no-ex-assign": 1, // Disallow reassigning exceptions in `catch` clauses
+		"no-extra-boolean-cast": 1, // Disallow unnecessary boolean casts
+		"no-fallthrough": 1, // Disallow fallthrough of `case` statements
+		"no-func-assign": 1, // Disallow reassigning `function` declarations
+		"no-global-assign": 1, // Disallow assignments to native objects or read-only global variables
+		"no-import-assign": 1, // Disallow assigning to imported bindings
+		"no-inner-declarations": 1, // Disallow variable or `function` declarations in nested blocks
```

Comment:
Not sure about this. 
a) Function declarations within function can sometimes be the cleaner solution
b) Hoisting is literally the only reason to use `var` right now. If anything, I'd welcome a rule that *only* allows `var` when used like this, and forces `let` when it would be equivalent to `var`.

Reply:
Removed this.